### PR TITLE
DE24092: volume: add export count attribute

### PR DESCRIPTION
**Pull request by achu-1612**
### Description:
Added a new volume attribute export count from the client to the schema.
Client release - https://github.com/HewlettPackard/hpegl-metal-client/releases/tag/v1.5.23
Refer pr for more details - https://github.com/HewlettPackard/hpegl-metal-client/pull/132

Note: As fields are populated by the backend, they are marked as computed which will make sure the resources are not affected when a re-apply is issued.

### Test:
Build the provider post the changes
Imported an existing volume to the state.
Validate the terraform state to check the newly added attributes
Screenshot 2024-09-23 135024

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -6,7 +6,7 @@ require (
 	github.com/golangci/golangci-lint v1.58.2
 	github.com/hashicorp/terraform-plugin-docs v0.19.4
 	github.com/hashicorp/terraform-plugin-sdk/v2 v2.34.0
-	github.com/hewlettpackard/hpegl-metal-client v1.5.22
+	github.com/hewlettpackard/hpegl-metal-client v1.5.23-0.20240923072040-5b07002ffb48
```
Comment (achu-1612) [inline go.mod:9]: https://github.com/HewlettPackard/hpegl-metal-client/pull/132 for client change.
Will update the actual tag post merging of this pr.

Comment (achu-1612) [reply go.mod]: Updated the pr with released client-https://github.com/HewlettPackard/hpegl-metal-client/releases/tag/v1.5.23

File: go.sum [hunk 2]
```diff
@@ -389,6 +389,10 @@ github.com/hashicorp/yamux v0.1.1 h1:yrQxtgseBDrq9Y652vSRDvsKCJKOUD+GzTS4Y0Y8pvE
 github.com/hashicorp/yamux v0.1.1/go.mod h1:CtWFDAQgb7dxtzFs4tWbplKIe2jSi3+5vKbgIO0SLnQ=
 github.com/hewlettpackard/hpegl-metal-client v1.5.22 h1:EW19Adpfroi27jpYYjlNddx/TwGKnTVysd5Lpqoo/ls=
 github.com/hewlettpackard/hpegl-metal-client v1.5.22/go.mod h1:+BO4BfQXf3IWnsxvk2IdE27Ijrm1jDVYIgpebYh1L9M=
+github.com/hewlettpackard/hpegl-metal-client v1.5.23-0.20240923072040-5b07002ffb48 h1:eaHwFgw9BaQn7O7AMchZgOBxfOygrP7lXKoAHUmsWEc=
+github.com/hewlettpackard/hpegl-metal-client v1.5.23-0.20240923072040-5b07002ffb48/go.mod h1:+BO4BfQXf3IWnsxvk2IdE27Ijrm1jDVYIgpebYh1L9M=
+github.com/hewlettpackard/hpegl-metal-client v1.5.23 h1:EQuYg4qGpUUF11hNYqb3ekzUKgqIfQd2RNg1Y3SCGqc=
+github.com/hewlettpackard/hpegl-metal-client v1.5.23/go.mod h1:+BO4BfQXf3IWnsxvk2IdE27Ijrm1jDVYIgpebYh1L9M=
```
Comment (neilgierman) [inline go.sum:395]: Did you `go mod tidy`? I am surprised to see v1.5.22 still in `go.sum`.

Comment (achu-1612) [reply go.sum]: Updated.

**Review by dbozzato81 — APPROVED** [review]
LGTM

**Review by harinathakotha — APPROVED** [review]
LGTM

**Pull request merged**